src/NUnitTestExtractor/NUnitTestExtractorApp.cs

```diff
@@ -16,383 +15,264 @@ public static class NUnitTestExtractorApp
     {
         private static Options _options = new Options();
 
-        public enum Level { Namespace, Class, Function, TestCase, Null }
+        private static Assembly _assembly = null;
 
-        static void Main(string[] args)
-        {
-            //Setting a default value
-            Level level = Level.Null;
+        public enum Level { Namespace, Class, Function, TestCase };
 
+        public static void Main(string[] args)
+        {
             using (var parser = new CommandLine.Parser(with => with.HelpWriter = Console.Error))
             {
                 if (parser.ParseArgumentsStrict(args, _options, () => Environment.Exit(-1)))
                 {
-                    if (_options.Help)
-                    {
-                        Console.WriteLine(CommandLine.Text.HelpText.AutoBuild(_options));
-                        return;
-                    }
-                   
-                    if(!string.IsNullOrEmpty(_options.Output))
+                    try
                     {
-                        string directory = null;
-
-                        int index = _options.Output.LastIndexOf("/", StringComparison.OrdinalIgnoreCase);
-
-                        if (index > 0)
+                        if (_options.Help)
                         {
-                            directory = _options.Output.Substring(0, index);
-                        }  
+                            Console.WriteLine(CommandLine.Text.HelpText.AutoBuild(_options));
 
-                        if (directory!=null && !Directory.Exists(directory))
-                        {
-                            Directory.CreateDirectory(directory);
+                            Environment.Exit(0);
                         }
 
-                        if (!File.Exists(_options.Output))
+                        // _options.Output is an abusolute path of output file for the test name list
```

Comment:
Typo: abusolute -> absolute

Reply:
This code is no longer existed in the latest version